wgpu-types/src/lib.rs

```diff
@@ -975,6 +975,8 @@ pub struct AdapterInfo {
     pub device_type: DeviceType,
     /// Backend used for device
     pub backend: Backend,
+    /// MPO(Multiple Plane Overlay) supported
+    pub mpo: bool,
```

Comment:
why does this flag need to be on adapterinfo? Is it applicable to other backends to some way? Is it actionable by the user?

Reply:
MPO is a hardware feature and not actionable by the user, if hardware supports MPO and the surface meets certain conditions, window compositor will display the surface as an MPO automatically.

Reply:
We can use [vkGetPhysicalDeviceDisplayPlanePropertiesKHR](https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/vkGetPhysicalDeviceDisplayPlanePropertiesKHR.html) to get the number of display planes available on `vulkan` backend.

Reply:
I started reading up on it and got lost a bit. So this is KHR_display: we have planes and displays, in addition to adapters and surfaces. wgpu currently doesn't have a way to expose planes or displays.

What value does this `mpo` flag bring to users? It seems like the actual API landscape is much more complicated than just a single boolean.

Reply:
In my scenario, I can suggest users to use the app in full screen mode based on this flag, which can effectively increase frame rate and reduce latency.

Reply:
ok, right. But even if the adapter itself supports MPO, it doesn't necessarily mean that the particular window/surface/adapter combination is going to support MPO, right?

Reply:
Yes, I thought about it, I can integrate this detection function in other tools, not in wgpu, so this PR will only include the create_surface_from_visual feature.

Reply:
that's a good way forward, thank you!
I'm totally positive about exposing some capability, it's just not clear what shape this would be, and we can follow-up on it.